spec/features/web_helpers.rb

```diff
@@ -0,0 +1,13 @@
+def sign_in_and_play
+    visit('/')
+    within("#name_form") do
+      fill_in 'player_name', with: 'stella'
+    end
+    click_button 'submit'
+end
+
+def play_move(move)
+  visit('/')
+  sign_in_and_play
+  click_button move
+end
```

Comment:
Really like the use of web helpers here to keep your code DRY. Excellent use of the within method to input player name.